src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -21,21 +21,23 @@ import org.broadinstitute.dsde.workbench.model.WorkbenchIdentityJsonSupport._
 import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport._
 import org.broadinstitute.dsde.workbench.model.google._
 import org.broadinstitute.dsde.workbench.model._
-import spray.json._
+import spray.json.{RootJsonFormat, RootJsonReader, _}
+import ca.mrvisser.sealerate
 
 // Create cluster API request
-case class ClusterRequest(labels: Option[LabelMap] = Option(Map.empty),
```

Comment:
Unless there's a reason to distinguish `None` and `Some(Map.emtpy)`, I think we should remove the `Option` wrapper here.

Reply:
there is, we don't want the field to be required in API requests. this would break backwards compatibility